doc/dev-jumpstart.md

```diff
@@ -26,6 +26,23 @@ not stable enough to be moved into `tests`.
 * The `scripts` folder contains several helper scripts for using Mystikos
 or integrating with the pipeline.
 
+Other directories include:
+
+* `apline`: contains Docker image used by some tests
+* `art`: contains graphical resources used in documentation
+* `asm`: contains assembly code
+* `crt`: contains Mystikos's modification to musl library
+* `doc`: contains documentation
+* `docker`: contains file to build Mystikos on Docker
+* `ext2`: contains EXT2 file system implementation
+* `gcov`: contains gcov code coverage tool
+* `host`: contains files used by Mystikos host
+* `hostfs`: a file system not in use
```

Comment:
Not being used is a bit too harsh :) We do use it in tests and it can be a good debugging tool(like the linux target).
Hostfs is an optional feature which allows mounting host directories either as rootfs or non-root mount in Mystikos. Users can opt-in for hostfs by building Mystikos with `MYST_ENABLE_HOSTFS` build flag. This flag is disabled for releases.